src/tests/test.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect, test } from "@playwright/test";
+import { LoginPage } from "../pages/Login.page";
+import { InventoryPage } from "../pages/Inventory.page";
+import { BasePage } from "../pages/Base.page";
+import { ShoppingCartPage } from "../pages/ShoppingCart.page";
+test("Sorting product items on Inventory page Z-A", async ({ page }) => {
```

Comment:
here better to add an empty line to separate test and imports

Reply:
As Pavlo mentioned url is present in  playwright.config.js